website/app/components/doc/related-components/index.hbs

```diff
@@ -6,4 +6,18 @@
 <hr class="doc-markdown-hr" />
 
 <h2 class="doc-text-h2">Related</h2>
-<Doc::Cards::Deck @cols="2" @layout="horizontal" @cards={{@cards}} />
\ No newline at end of file
+
+<Doc::Cards::Deck @cols="2">
+  {{#each @cards as |card|}}
+    <Doc::Cards::Card
+      @image={{card.image}}
+      @title={{card.title}}
+      @caption={{card.caption}}
+      @route={{card.route}}
+      @model={{card.model}}
+      @href={{card.href}}
+      @layout="horizontal"
+      {{doc-track-event-on "click" "Related - {{@title}} - {{card.title}}"}}
```

Comment:
Question for @Dhaulagiri @heatherlarsen: what are we trying to measure here?
- if people click on the "related" components?
- what of the "related components" are clicked, in which page?

If it was me, I would measure only the first, because the second one would be **very** fragmented and probably we would not see any meaningful signal

Reply:
That could be. I think the idea was that we could see which related components are being clicked from which parent component to see which combinations were being used the most.